src/Language/ASKEE/Exposure/Interpreter.hs

```diff
@@ -124,10 +130,13 @@ suspend v
 data EvalRead = EvalRead
   { erPrecomputed :: Map Value Value
   , erLocalVars   :: Map Ident Value
+  , putFileFn     :: FilePath -> LBS.ByteString -> IO ()
+  , getFileFn     :: FilePath -> IO LBS.ByteString
   }
+   -- TODO: Should the IO be LBS? DataSeries requires it for read, and why not be consistent...?
```

Comment:
I'm not sure I understand what this TODO is saying. The code above is already using `LBS`, right?